src/Microsoft.AspNetCore.Mvc.Core/DependencyInjection/MvcCoreServiceCollectionExtensions.cs

```diff
@@ -22,8 +22,16 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    /// <summary>
+    /// Extension methods for setting up essential MVC services in an <see cref="IServiceCollection" />.
+    /// </summary>
     public static class MvcCoreServiceCollectionExtensions
     {
+        /// <summary>
+        /// Adds essential MVC services to the specified <see cref="IServiceCollection" />.
```

Comment:
As discussed [here](https://github.com/aspnet/Mvc/pull/4159#discussion-diff-54208453), ignore this comment too. `services` is a `this` parameter.